app/assets/stylesheets/expenses.css.scss

```diff
@@ -2,11 +2,16 @@
   float: right;
 }
 
-#create_expense_link_text {
+#expenses_page_heading {
+  margin-right: 135px;
+  margin-top: 40px;
+}
+
+#calculate_total_expenses_link_text {
+  margin-left: 9px;
   margin-top: 36px;
 }
 
-#expenses_page_heading {
-  margin-right: 336px;
-  margin-top: 40px;
+#create_expense_link_text {
```

Comment:
Avoid using id selectors<br>Selector `create_expense_link_text` should be written in lowercase with hyphens